sdk/eventhub/event-hubs/test/eventProcessor.spec.ts

```diff
@@ -392,9 +407,9 @@ describe("Event Processor", function(): void {
       // pick up an extra surprise partition
       //
       // This particular behavior is really specific to the FairPartitionLoadBalancer but that's okay for now.
```

Comment:
I don't think the comment above is accurate anymore (perhaps the reasoning changed). Can you update it to what '5' represents now?

Reply:
Updated!